### DISCUSSION TITLE
Is this repo is still maintained

### DISCUSSION THREAD

**Discussion by VIP-IOIO** [NetrisTV/ws-scrcpy#303] [General]
Hi @kkuehl, @drauggres, @BiteFoo, @lizilong-1996

just wanted to check, if this project is still maintained or not by the developer, since last release was 2022.

Thank you for your valuable contribution to open source community, but can you please confirm.

**Comment by drauggres** [comment]
Hi.

I no longer work at @NetrisTV. Also I don't use this project for personal purposes and I don't really have time to activelly maintain it.
So I only work on critical build problems and security issues.
